shard.yml

```diff
@@ -6,7 +6,7 @@ authors:
 
 license: MIT
 
-crystal: 0.23.1
+crystal: 0.24.0
```

Comment:
should this be 0.24.1?

Reply:
The breaking change was in 0.24.0 but it’s almost the same to use .0 or .1 since 0.24.0 could be treated as a yanked release.

Reply:
@bcardiff @mosop I don't have permission to merge this